src/iterator/raw.rs

```diff
@@ -49,7 +49,7 @@ pub struct RawDataSlice<'a, R, BO> {
 
 impl<'a, R, BO> RawDataSlice<'a, R, BO> {
     /// Creates a new raw data slice.
-    pub fn new(data: &'a [u8]) -> Self {
+    pub const fn new(data: &'a [u8]) -> Self {
```

Comment:
I'm not really familiar with how this is used. Maybe there's no point to making this one `const`?